spec/support/gemfiles/Gemfile.rails-5.2.x

```diff
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+
+gemspec :path => '../../../'
+
+gem 'rails', '~> 5.2.0'
+
+# Rails imposed mysql2 version contraints
+# https://github.com/rails/rails/blob/5-2-stable/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb#L4
```

Comment:
Line 6 seems to the appropriate line. (`#6`) Could you please change the comment?

Reply:
@orien sorry I do not quite get what you mean

Reply:
If you follow this link it highlights the wrong line:
https://github.com/rails/rails/blob/5-2-stable/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb#L4